src/main/java/cc/hyperium/cosmetics/AbstractCosmetic.java

```diff
@@ -20,14 +19,15 @@
     private final EnumPurchaseType purchaseType;
     private final Map<UUID, Boolean> purchasedBy = new ConcurrentHashMap<>();
     private boolean selfUnlocked;
+    private boolean checkDevEnv;
 
     public AbstractCosmetic(boolean selfOnly, EnumPurchaseType purchaseType) {
         this.selfOnly = selfOnly;
         this.purchaseType = purchaseType;
         try {
             PurchaseApi.getInstance().getPackageAsync(UUIDUtil.getClientUUID(), hyperiumPurchase -> {
                 if (hyperiumPurchase == null) {
-                    System.out.println("WARNING COSMETIC NULL");
+                    System.out.println("[Cosmetics] Detected " + getPurchaseType().toString().toLowerCase() + " is not purchased!");
```

Comment:
This should simply say the cosmetic is null. It might actually be purchased but be null for another reason for example.

Reply:
Okay